content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -15,59 +15,80 @@ This document describes the concept of a StorageClass in Kubernetes. Familiarity
+storage system designs.
+
+<!-- body -->
+
+## StorageClass objects
 
 Each StorageClass contains the fields `provisioner`, `parameters`, and
 `reclaimPolicy`, which are used when a PersistentVolume belonging to the
-class needs to be dynamically provisioned.
+class needs to be dynamically provisioned to satisfy a PersistentVolumeClaim (PVC).
 
 The name of a StorageClass object is significant, and is how users can
 request a particular class. Administrators set the name and other parameters
 of a class when first creating StorageClass objects.
 
-Administrators can specify a default StorageClass only for PVCs that don't
-request any particular class to bind to: see the
-[PersistentVolumeClaim section](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)
-for details.
+As an administrator, you can specify a default StorageClass that applies to any PVCs that
+don't request any particular class to bind to; see the
+[PersistentVolumeClaim concept](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)
+for more details.
 
-```yaml
-apiVersion: storage.k8s.io/v1
-kind: StorageClass
-metadata:
-  name: standard
-provisioner: kubernetes.io/aws-ebs
-parameters:
-  type: gp2
-reclaimPolicy: Retain
-allowVolumeExpansion: true
-mountOptions:
-  - debug
-volumeBindingMode: Immediate
-```
+Here's an example of a StorageClass:
 
-### Default StorageClass
+{{% code file="storage/storageclass-low-latency.yaml" %}}
 
-When a PVC does not specify a `storageClassName`, the default StorageClass is
-used. The cluster can only have one default StorageClass. If more than one
-default StorageClass is accidentally set, the newest default is used when the
-PVC is dynamically provisioned.
+## Default StorageClass
 
+You can mark a StorageClass as the default for your cluster.
 For instructions on setting the default StorageClass, see
 [Change the default StorageClass](/docs/tasks/administer-cluster/change-default-storage-class/).
-Note that certain cloud providers may already define a default StorageClass.
 
-### Provisioner
+When a PVC does not specify a `storageClassName`, the default StorageClass is
+used.
+
+If you set the
+[`storageclass.kubernetes.io/is-default-class`](/docs/reference/labels-annotations-taints/#ingressclass-kubernetes-io-is-default-class)
+annotation to true on more than one StorageClass in your cluster, and you then
+create a PersistentVolumeClaim with no `storageClassName` set, Kubernetes
+picks which of the “default” StorageClasses to use based on which was created
+more recently.
+
+{{< note >}}
+You should try to only have one StorageClass in your cluster that is
+marked as the default. The reason that Kubernetes allows you to have
+two (or more) default StorageClasses is to allow for seamless migration.
+{{< /note >}}
+
+You can create a PersistentVolumeClaim without specifying a `storageClassName`
+for the new PVC, and you can do so even when no default StorageClass exists
```

Comment:
```suggestion
, and you can do so even when no default StorageClass exists
```

Reply:
I prefer the original - what's not right with it?